ruby/.rubocop.yml

```diff
@@ -1,12 +1,43 @@
-# Commonly used screens these days easily fit more than 80 characters.
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - "bin/*"
+    - "config/**/*"
```

Comment:
**Update**

After actually pushing changes to Github, some failures appeared that doesn't appear in the local environments. 

This happened to both repositories.

They are all problems with the **following Cop**:

```yml
C: Layout/ExtraSpacing: Unnecessary spacing detected.
```

And all happen in:

```yml
config/****.rb
```

and

```yml
config/environments/**.rb
```

**Quick fix:**
delete line 5, running automatic correction, then add it back or not.
It seems to block line 37 in the local environment for the config folder! but allow it for Stickler bot!

* Deleting rubocob.yml temporrely and performing auto-correction with no exceptions is not a bad Idea in general!

Reply:
So, I forked your micro-reddit repo and:

- bad news - I found out that Stickler ignores `exclude` in .rubocop.yml :)
- good news - there is an option to exclude files in .stickler.yml

I made sure they behave in the same way. Thanks for report!